src/main/java/io/cryostat/recordings/JvmIdHelper.java

```diff
@@ -140,7 +152,7 @@ public ServiceRef resolveId(ServiceRef sr) throws JvmIdGetException {
             updated.setCryostatAnnotations(sr.getCryostatAnnotations());
             return updated;
         } catch (InterruptedException | ExecutionException | TimeoutException | ScriptException e) {
-            logger.warn("Could not get jvmId for target {}", uriStr);
+            logger.warn("Could not resolve jvmId for target {}", uriStr);
```

Comment:
I realize this doesn't actually need to be here, because `DiscoveryStorage#modifyChildrenWithJvmIds` already warns, so one of these can be removed.